src/client/commands/submit.rs

```diff
@@ -22,6 +22,7 @@ use crate::transfer::messages::{
 };
 use crate::{rpc_call, JobId, JobTaskCount};
 
+const SUBMIT_ARRAY_LIMIT: JobTaskCount = 10000;
```

Comment:
I would lower this, at least to `1000` or even `100`. @spirali?

Reply:
It is really hard question :) I would stick with 1000 (or 999 rather so it will triggered at 1000). But I can imagine that even 100 is a good limit for some users

Reply:
Ok let's go with `999`.